packages/cheerio-crawler/src/internals/cheerio-crawler.ts

```diff
@@ -172,25 +173,21 @@ export class CheerioCrawler extends HttpCrawler<CheerioCrawlingContext> {
         isXml: boolean,
         crawlingContext: CheerioCrawlingContext,
     ) {
-        const dom = await this._parseHtmlToDom(response, isXml);
```

Comment:
this is no longer used anywhere now. lets add a TODO to that method to remove it in v4, doing so now is technically breaking, people might be extending the CheerioCrawler.

Reply:
Yup, I was thinking about whether removing this is the acceptable amount of breaking change (I cannot imagine anyone calling this one method directly). Let's stay on the safe side - I'll add the todo comment.